EventFilter/CSCRawToDigi/plugins/DigiAnalyzer.cc

```diff
@@ -30,7 +30,7 @@
 #include "DataFormats/CSCDigi/interface/CSCWireDigi.h"
 #include "DataFormats/CSCDigi/interface/CSCStripDigi.h"
 
-class DigiAnalyzer : public edm::EDAnalyzer {
+class DigiAnalyzer : public edm::one::EDAnalyzer<> {
```

Comment:
At some point (not necessarily in this PR) it could be worth renaming this class as `CSCDigiAnalyzer`, in accordance to what it done in similar analyzers from other subsystems